### ISSUE TITLE
NullExeption in ValueSet.Add(string, object)

### ISSUE THREAD

**Issue by Marv51** [microsoft/CsWinRT#494]
On 
```
dotnet --version
5.0.100-rc.2.20479.15
```
I create a new wpf project `dotnet new wpf` and change the `TargetFramework` to `net5.0-windows10.0.19041.0`

In the MainWindow constructor i have added
```
 var p = new Windows.Foundation.Collections.ValueSet();
 p.Add("test", "test");
```

this throws a NullReferenceException (see below for details). This is a regression and used to work in 3.1.

As a workaround the (less elegant) version: `p.Add(new KeyValuePair<string, object>("test", "test"))` works.

---
System.NullReferenceException
  HResult=0x80004003
  Message=Object reference not set to an instance of an object.
  Source=WinRT.Runtime
  StackTrace:
   at ABI.System.Collections.Generic.IDictionary`2.FromAbiHelper.Insert(IMap`2 _this, K key, V value)
   at ABI.System.Collections.Generic.IDictionary`2.FromAbiHelper.Add(K key, V value)
   at ABI.System.Collections.Generic.IDictionary`2.global::System.Collections.Generic.IDictionary<K,V>.Add(K key, V value)
   at Windows.Foundation.Collections.ValueSet.Add(String key, Object value)
   at valueset.MainWindow..ctor() in C:\Users\xxx\source\repos\valueset\MainWindow.xaml.cs:line 27

**Comment by AdamBraden** [comment]
@ujjwalchadha can you confirm if this is fixed by #488

**Comment by ujjwalchadha** [comment]
Seems to be fixed by #488
